packages/@stylexjs/stylex/src/types/StyleXCSSTypes.js

```diff
@@ -100,7 +102,7 @@ type alignSelf =
   | 'safe center'
   | 'unsafe center'
   | all;
-type all = null | 'initial' | 'inherit' | 'unset';
+type all = null | 'initial' | 'inherit' | 'unset' | CSSType;
```

Comment:
[`all` is a CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/all). So this change doesn't looks right

Reply:
Oh - I missed the intention here definitely. Any clues where to put it best so it’s allowed everywhere? I’ll also take a look at it tomorrow. Thanks!

I also see the test failing - missed it previously, will resolve that as well

Reply:
Hello 👋

Changed that approach to one that I'm a lot more confident about - there was already a type helper `ComplexStyleValueType` which when stuff like `StyleXVar` was used inside of `stylex.create` it resolved it to the underlaying value at the type level. The logic of resolving `CSSType` fits perfectly here imho. Wdyt?